apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.spec.ts

```diff
@@ -212,6 +233,7 @@ describe('DotEditContentComponent', () => {
                 DotOverlayMaskModule,
                 DotWizardModule,
                 DotContentPaletteModule,
+                DotIconModule,
```

Comment:
did you check on all the new tests that you're not introducing any new error:
`ERROR: 'NG0304: 'dot-icon' is not a known element:`
???

Reply:
![Uploading image.png…]()